pipeline/lookup.csv

```diff
@@ -29678,3 +29678,921 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
+brownfield-land,,local-authority-eng:EST,ESBC-BR69,1730511
+brownfield-land,,local-authority-eng:EST,ESBC-BR70,1730512
+brownfield-land,,local-authority-eng:EST,ESBC-BR71,1730513
+brownfield-land,,local-authority-eng:EST,ESBC-BR72,1730514
+brownfield-land,,local-authority-eng:EST,ESBC-BR73,1730515
+brownfield-land,,local-authority-eng:EST,ESBC-BR74,1730516
+brownfield-land,,local-authority-eng:EST,ESBC-BR75,1730517
+brownfield-land,,local-authority-eng:EST,ESBC-BR76,1730518
+brownfield-land,,local-authority-eng:EST,ESBC-BR77,1730519
+brownfield-land,,local-authority-eng:ESK,ES086,1730520
+brownfield-land,,local-authority-eng:ESK,ES087,1730521
+brownfield-land,,local-authority-eng:ESK,ES088,1730522
+brownfield-land,,local-authority-eng:ESK,ES089,1730523
+brownfield-land,,local-authority-eng:ESK,ES090,1730524
+brownfield-land,,local-authority-eng:ESK,ES091,1730525
+brownfield-land,,local-authority-eng:ESK,ES092,1730526
+brownfield-land,,local-authority-eng:ESK,ES093,1730527
+brownfield-land,,local-authority-eng:ERE,BLR74,1730528
+brownfield-land,,local-authority-eng:ERE,BLR59,1730529
+brownfield-land,,local-authority-eng:ERE,BLR66,1730530
+brownfield-land,,local-authority-eng:ERE,BLR67,1730531
+brownfield-land,,local-authority-eng:ERE,BLR60,1730532
+brownfield-land,,local-authority-eng:ERE,BLR76,1730533
+brownfield-land,,local-authority-eng:ERE,BLR57,1730534
+brownfield-land,,local-authority-eng:ERE,BLR70,1730535
+brownfield-land,,local-authority-eng:ERE,BLR62,1730536
+brownfield-land,,local-authority-eng:ERE,BLR58,1730537
+brownfield-land,,local-authority-eng:ERE,BLR71,1730538
+brownfield-land,,local-authority-eng:ERE,BLR75,1730539
+brownfield-land,,local-authority-eng:ERE,BLR72,1730540
+brownfield-land,,local-authority-eng:ERE,BLR79,1730541
+brownfield-land,,local-authority-eng:ERE,BLR55,1730542
+brownfield-land,,local-authority-eng:ERE,BLR73,1730543
+brownfield-land,,local-authority-eng:ERE,BLR81,1730544
+brownfield-land,,local-authority-eng:ERE,BLR69,1730545
+brownfield-land,,local-authority-eng:ERE,BLR65,1730546
+brownfield-land,,local-authority-eng:ERE,BLR61,1730547
+brownfield-land,,local-authority-eng:ERE,BLR64,1730548
+brownfield-land,,local-authority-eng:ERE,BLR80,1730549
+brownfield-land,,local-authority-eng:ERE,BLR78,1730550
+brownfield-land,,local-authority-eng:ERE,BLR77,1730551
+brownfield-land,,local-authority-eng:ERE,BLR68,1730552
+brownfield-land,,local-authority-eng:ERE,BLR56,1730553
+brownfield-land,,local-authority-eng:ERE,BLR63,1730554
+brownfield-land,,local-authority-eng:FAR,3247,1730555
+brownfield-land,,local-authority-eng:FAR,3253,1730556
+brownfield-land,,local-authority-eng:FAR,3255,1730557
+brownfield-land,,local-authority-eng:FAR,3257,1730558
+brownfield-land,,local-authority-eng:FAR,3259,1730559
+brownfield-land,,local-authority-eng:FAR,3260,1730560
+brownfield-land,,local-authority-eng:FAR,3263,1730561
+brownfield-land,,local-authority-eng:FAR,3264,1730562
+brownfield-land,,,,1730563

```

Comment:
These lines are incorrect, they cannot be blank for organisation and reference

Reply:
I've removed the incorrect lines from lookup.csv in change 102-114 linked to this PR request